src/imitation/data/rollout.py

```diff
@@ -419,7 +431,9 @@ def generate_trajectories(
     #
     # To start with, all environments are active.
     active = np.ones(venv.num_envs, dtype=bool)
-    assert isinstance(obs, np.ndarray), "Dict/tuple observations are not supported."
+    if not isinstance(obs, np.ndarray):
+        raise ValueError("Dict/tuple observations are not supported.")
```

Comment:
Prefer:
```
"Dict/tuple observations are not supported. Currently only np.ndarray observations are supported."
```

Only mentioning the first kind leaves open the option that we might be currently supporting something other than np.ndarray. I think that the only options are dict/tuple/np.ndarray so this may be slightly pedantic, but 🤷

Reply:
You are right, I am also a fan of actionable error messages!